packages/katello/katello/katello.spec

```diff
@@ -5,7 +5,7 @@
 %global confdir common
 %global prereleasesource master
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
-%global release 7
+%global release 8
 
 Name:       katello
 Version:    4.0.0
```

Comment:
Aren't we at 4.1.0 by now?

Reply:
I think folks are working on that, but this PR is not the one to change the version. And this change applies to 4.0 as well.